page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/data/DataProvider.java

```diff
@@ -0,0 +1,57 @@
+package ru.sbtqa.tag.pagefactory.data;
+
+import static java.lang.String.*;
+import ru.sbtqa.tag.datajack.TestDataObject;
+import ru.sbtqa.tag.datajack.adaptors.ExcelDataObjectAdaptor;
+import ru.sbtqa.tag.datajack.adaptors.JsonDataObjectAdaptor;
+import ru.sbtqa.tag.datajack.adaptors.PropertiesDataObjectAdaptor;
+import ru.sbtqa.tag.datajack.exceptions.DataException;
+import ru.sbtqa.tag.qautils.properties.Props;
+
+public class DataProvider {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=64&issues=AWSn0wiesk71k7lw7bXU&open=AWSn0wiesk71k7lw7bXU)

[ISSUE_KEY]: AWSn0wiesk71k7lw7bXU